templates/taxi/car_detail.html

```diff
@@ -11,12 +11,23 @@
       Update
     </a>
   </h1>
+  <h2>Drivers</h2>
+  <form method="post" action="{% url 'taxi:car-add-or-delete-driver' car.id %}">
+    {% csrf_token %}
+    {% if user in car.drivers.all %}
+      <input type="submit" value="Delete me from this car" class="btn btn-danger link-to-page">
+    {% else %}
+      <input type="submit" value="Assign me to this car" class="btn btn-success link-to-page">
+    {% endif %}
```

Comment:
The logic for checking if the user is in `car.drivers.all` is correct. However, ensure that the `user` context variable is always available in the template, as it is critical for this conditional logic to work.